src/catch2/catch_config.cpp

```diff
@@ -113,6 +113,25 @@ namespace Catch {
     bool Config::listReporters() const      { return m_data.listReporters; }
     bool Config::listListeners() const      { return m_data.listListeners; }
 
+    bool Config::hasBazel() const {
+#ifdef CATCH_CONFIG_BAZEL_SUPPORT
+        return true;
+#endif
```

Comment:
This will cause annoying warning issues about unreachable code. The simplest solution, albeit a bit annoying, is to turn this into an `#else` and then have the code below also be conditionally compiled.